modules/extension/xsd/xsd-fes/src/main/java/org/geotools/filter/LikeFilterFesImpl.java

```diff
@@ -0,0 +1,53 @@
+package org.geotools.filter;
+
+import org.opengis.filter.expression.Expression;
+
+/**
+ * An implementation of {@link org.opengis.filter.PropertyIsLike} that supports expressions on the
+ * right hand side, in accordance to the Filter 2.0 schema. Permits visitors to traverse a function
+ * chain if necessary.
+ */
+public class LikeFilterFesImpl extends LikeFilterImpl {
```

Comment:
I didn't think this impl was ready for the `library` module quite yet. If the `xsd-fes` module is included in the project, this class should show up under the same package anyway.